tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php

```diff
@@ -69,6 +69,53 @@ public function providerInvalidConfigurationElement()
         );
     }
 
+    public function testConfigureResetsExclude()
```

Comment:
this test method is checking that `null` is causing a reset of configuration for this one Fixer class.
what about making it general, to run on all Fixers ?